### ISSUE TITLE
Training on a new dataset based on best.pt

### ISSUE THREAD

**Issue by hzwhl** [ultralytics/ultralytics#14420]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello, thank you for your continued work. I have a question about training: If I have trained the best.pt model on a dataset, I want to further improve the accuracy of the best.pt model based on another new dataset. Can I just use the best.pt model as the pre-trained model for the new dataset? For example, use the following code:

```python
from ultralytics import YOLO
if __name__ == '__main__':
# Load a model
    model = YOLO("best.pt")  
# Use the model
    model.train(data="config.yaml", patience=15, epochs=500, batch=20, imgsz=1280)  
```

Here config.yaml points to the new dataset.

**Comment by hzwhl** [comment]
Also, what difference does setting pretrained to False or True make after using the best.pt model obtained previously, for example:

from ultralytics import YOLO
if __name__ == '__main__':
    model = YOLO("New_4.0.2_1024_1280.pt")  
    model.train(data="config.yaml", patience=15, epochs=500, batch=20, imgsz=1280, pretrained=True) 

or

from ultralytics import YOLO
if __name__ == '__main__':
    model = YOLO("New_4.0.2_1024_1280.pt")  
    model.train(data="config.yaml", patience=15, epochs=500, batch=20, imgsz=1280, pretrained=False)

**Comment by Y-T-G** [comment]
`pretrained=True` or False doesn't do anything.

For fine-tuning, you should pass lower learning rate, specify optimizer and disable warmup epochs. `lr0=0.001, optimizer="SGD", warmup_epochs=0`

**Comment by glenn-jocher** [comment]
@hzwhl @Y-T-G ah actually the `pretrained` argument is supposed to remove any trained weights from a model and leave it with a randomly initialized architecture so that basically these two commands would be identical:

```
yolo train model=yolov8n.yaml
yolo train model=yolov8n.pt pretrained=False
```

**Comment by hzwhl** [comment]
@glenn-jocher 
Hello, sorry to bother you again, what is the difference between the following three commands：

yolo train model=yolov8n.yaml
yolo train model=yolov8n.pt pretrained=False
yolo train model=yolov8n.pt pretrained=True

My understanding is that there is no difference between the first and second commands, and the third command means to continue training based on the existing weight parameters of the yolov8n.pt model. Is this correct?

**Comment by glenn-jocher** [comment]
@hzwhl that should be correct, though you may want to test to verify. Pretrained weights will show much higher mAP on epoch 1, you can test with i.e. `yolo train data=coco8.yaml`, whereas randomly initialized weights will show mAP=0.

**Comment by Y-T-G** [comment]
@glenn-jocher I see. From examining the code, I thought pretrained serves the same functionality as .`load()` in the API. So you are required to provided a path to a checkpoint at it will load the weights from there. I can see it's ignored otherwise here (when it's simply a boolean and not a path):
https://github.com/ultralytics/ultralytics/blob/e094f9c3718e3e43f17299b9919da696d4b96887/ultralytics/engine/trainer.py#L545-L546

But I can see the boolean values do have an effect on classification models:
https://github.com/ultralytics/ultralytics/blob/e094f9c3718e3e43f17299b9919da696d4b96887/ultralytics/models/yolo/classify/train.py#L51-L52
https://github.com/ultralytics/ultralytics/blob/e094f9c3718e3e43f17299b9919da696d4b96887/ultralytics/models/yolo/classify/train.py#L65

I can't find the code where it's resetting weight for non-classification models.

**Comment by glenn-jocher** [comment]
Hmm strange, maybe this is not implemented for detection models, as at first this arg was only for classify models which use a different dataloader.

Another workflow that can be done too that's related is to create a blank model and then transfer weights from any other model (only layers with matching shapes will transfer):

```python
model = YOLO('path/to/model.yaml').load('any/other/model.pt')
```
